### ISSUE TITLE
Feature request - "fuck" for "fuck" that's misspelled 

### ISSUE THREAD

**Issue by GildedHonour** [nvbn/thefuck#1065]
If I say fruck or fuk, I want to say "fuck" and it should give me the corrent version of the first fuck.

**Comment by alepinio** [comment]
ok but, next fuck should skip first fuck in order to the point to the first mispelled command?

**Comment by alepinio** [comment]
ok but, next fuck should skip first fuck in order to point to the first mispelled command?

**Comment by GildedHonour** [comment]
That's up to fuck.

El 19 de abril de 2020 20:42:20 GMT+05:30, alepinio <notifications@github.com> escribió:
>ok but, next fuck should skip first fuck in order to point to the first
>mispelled command?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nvbn/thefuck/issues/1065#issuecomment-616158281

-- 
Enviado desde mi dispositivo Android con K-9 Mail. Por favor, disculpa mi brevedad.

**Comment by brunodantas** [comment]
I was also wondering if it could fuck itself.
Surely an essential feature.

**Comment by Joe-Sin7h** [comment]
Yes , it'll be a very essential feature.

**Comment by scorphus** [comment]
What if, for instance, you're trying to unfuck a mispelled `fsck`?

**Comment by scorphus** [comment]
Accidentally closed 🤦🏻‍♂️

**Comment by leostudiooo** [comment]
May be like this:
```
$ git pul
git: 'pul' is not a git command. See 'git --help'.

The most similar commands are
        pull
        push

$ fuk
bash: No such file or directory.

$ fuck
fuck [enter/↑/↓/ctrl+c]
git pull [enter/↑/↓/ctrl+c]
Already up-to-date.
```

**Comment by scorphus** [comment]
Depending on your shell, this is what you get. Another example:

```
$ git dif
git: 'dif' is not a git command. See 'git --help'.

The most similar commands are
	diff
	df
$ fuk
bash: fuk: command not found
$ fuck
git diff [enter/edit/↑/↓/ctrl+c]
```

**Comment by divykj** [comment]
@scorphus In my case, thefuck is fucking itself over and over again 😕
![image](https://user-images.githubusercontent.com/62170586/103151225-240c6c00-47a2-11eb-86b4-ec43656701fe.png)

**Comment by scorphus** [comment]
Yup. IIRC, we still need to properly set and export the `TF_HISTORY` variable in the Fish function.
